packages/twenty-server/src/modules/reports/standard-objects/analytics-query.workspace-entity.ts

```diff
@@ -0,0 +1,113 @@
+import { Relation } from 'src/engine/workspace-manager/workspace-sync-metadata/interfaces/relation.interface';
+
+import { FieldMetadataType } from 'src/engine/metadata-modules/field-metadata/field-metadata.entity';
+import {
+  RelationMetadataType,
+  RelationOnDeleteAction,
+} from 'src/engine/metadata-modules/relation-metadata/relation-metadata.entity';
+import { ANALYTICS_QUERY_STANDARD_FIELD_IDS } from 'src/engine/workspace-manager/workspace-sync-metadata/constants/standard-field-ids';
+import { STANDARD_OBJECT_IDS } from 'src/engine/workspace-manager/workspace-sync-metadata/constants/standard-object-ids';
+import { WorkspaceEntity } from 'src/engine/twenty-orm/decorators/workspace-entity.decorator';
+import { WorkspaceField } from 'src/engine/twenty-orm/decorators/workspace-field.decorator';
+import { WorkspaceRelation } from 'src/engine/twenty-orm/decorators/workspace-relation.decorator';
+import { WorkspaceIsNullable } from 'src/engine/twenty-orm/decorators/workspace-is-nullable.decorator';
+import { BaseWorkspaceEntity } from 'src/engine/twenty-orm/base.workspace-entity';
+import { AnalyticsQueryFilterWorkspaceEntity } from 'src/modules/reports/standard-objects/analytics-query-filter.workspace-entity';
+import { ChartWorkspaceEntity } from 'src/modules/reports/standard-objects/chart.workspace-entity';
+import { WorkspaceJoinColumn } from 'src/engine/twenty-orm/decorators/workspace-join-column.decorator';
+
+@WorkspaceEntity({
+  standardId: STANDARD_OBJECT_IDS.analyticsQuery,
+  namePlural: 'analyticsQueries',
+  labelSingular: 'Analytics Query',
+  labelPlural: 'Analytics Queries',
+  description: 'A query for analytics',
+  icon: 'IconDatabaseSearch',
+})
+export class AnalyticsQueryWorkspaceEntity extends BaseWorkspaceEntity {
+  @WorkspaceRelation({
+    standardId: ANALYTICS_QUERY_STANDARD_FIELD_IDS.chart,
+    label: 'Chart',
+    description: 'The chart this analytics query belongs to',
+    icon: 'IconChartBar',
+    type: RelationMetadataType.MANY_TO_ONE,
+    inverseSideTarget: () => ChartWorkspaceEntity,
+    inverseSideFieldKey: 'analyticsQueries',
+    onDelete: RelationOnDeleteAction.SET_NULL,
+  })
+  @WorkspaceIsNullable()
+  chart: Relation<ChartWorkspaceEntity> | null;
+
+  @WorkspaceJoinColumn('chart')
+  chartId: string | null;
```

Comment:
One chart can have multiple queries but a query cannot have multiple charts? I would have assume it would be the opposite. I'm not sure we need Chart object at all for a v1 tbh, it could be 1 query = 1 chart